sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/AnomalyDetection/SmartDetectionCondition.cs

```diff
@@ -1,22 +1,43 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
+using Azure.Core;
+
 namespace Azure.AI.MetricsAdvisor.Models
 {
+    // TODODOCS: properties need clarification.
     /// <summary>
     /// </summary>
     public partial class SmartDetectionCondition
     {
         /// <summary>
+        /// Creates a new instance of the <see cref="HardThresholdCondition"/> class.
         /// </summary>
-        public AnomalyDetectorDirection AnomalyDetectorDirection { get; }
+        /// <param name="sensitivity"> sensitivity, value range : (0, 100]. </param>
+        /// <param name="anomalyDetectorDirection"> detection direction. </param>
```

Comment:
capitalization

Reply:
Marked the whole class with `TODODOCS`. These are not done yet.